CHANGELOG.md

```diff
@@ -0,0 +1,179 @@
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+# Changelog
+
+- [Changelog](#changelog)
+  - [3.11.0](#3110)
+    - [New Features](#new-features)
+    - [Bug Fixes](#bug-fixes)
+  - [Documentation](#documentation)
+  - [Development](#development)
+  - [3.10.0](#3100)
+    - [New Features](#new-features-1)
+    - [Bug Fixes](#bug-fixes-1)
+  - [Documentation](#documentation-1)
+  - [Development](#development-1)
+  - [Previous versions](#previous-versions)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+# Changelog
+
+<!--
+
+This should be updated on every PR.
+
+We copy from here into the release notes.
+
+ -->
+
+<!--
+Add next version above previous version but below this line using the template
+
+## 0.0.0
+
+v0.0.0
+
+VS Code v0.00.0
+
+Upgrading is as easy as installing the new version over the old one. code-server
+maintains all user data in \`~/.local/share/code-server\` so that it is preserved in between
+installations.
+
+### New Features
+
+⭐ Summarize new features here with references to issues
+
+- item
+
+### Bug Fixes
+
+⭐ Summarize bug fixes here with references to issues
+
+- fix(socket): did this thing #321 @githubuser
+
+## Documentation
+
+⭐ Summarize doc changes here with references to issues
+
+- item
+
+## Development
+
+⭐ Summarize development/testing changes here with references to issues
+
+- item
+
+-->
+
+## 3.11.0
+
+v3.11.0
+
+VS Code v1.56
+
+Upgrading is as easy as installing the new version over the old one. code-server
+maintains all user data in \`~/.local/share/code-server\` so that it is preserved in between installations.
+
+### New Features
+
+⭐ Summarize new features here with references to issues
+
+- item
+
+### Bug Fixes
+
+⭐ Summarize bug fixes here with references to issues
+
+- fix(socket): did this thing #321 @githubuser
+
+## Documentation
+
+⭐ Summarize doc changes here with references to issues
+
+- item
+
+## Development
+
+⭐ Summarize development/testing changes here with references to issues
+
+- item
```

Comment:
Is the stub example for 3.11 (which is not commented out) intentional? I think it might be confusing given that there's a commented-out template right above it.

Reply:
Well I wasn't sure if between now and later today, someone might merge a PR that's meant for 3.11 but I guess that won't happen. I'll remove it!